WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationContentRouter.swift

```diff
@@ -59,7 +59,7 @@ struct NotificationContentRouter {
             try coordinator.displayCommentsWithPostId(notification.metaPostID,
                                                       siteID: notification.metaSiteID,
                                                       commentID: commentID,
-                                                      source: .commentNotification)
+                                                      source: .commentLikeNotification)
```

Comment:
I'm seeing `.commentLikeNotification` source when tapping on a comment notification. Since the case `.comment` just above this is `fallthrough` – should it be tracked with the `.commentNotification` source?

Reply:
Ah, I missed that. Thanks!